scripts/extensions/pool_update.precheck

```diff
@@ -49,16 +50,25 @@ def parse_control_package(session, yum_url):
     return items[0].getAttribute('control')
 
 
-def execute_precheck(session, control_package, yum_conf_file, nogpgcheck):
-    FNULL = open(os.devnull, 'w')
-    retcode = subprocess.call('yum -c ' + yum_conf_file + ' install -y ' +
-                              nogpgcheck + control_package, shell=True, stdout=FNULL, stderr=FNULL)
-    if retcode != 0:
+def execute_precheck(session, control_package, yum_conf_file):
+    cmd = ['yum', 'install', '-y', '--noplugins', '-c', yum_conf_file, control_package]
+    p = subprocess.Popen(cmd, shell=False, stdout=subprocess.PIPE, stderr=subprocess.STDOUT, close_fds=True)
+    p.wait()
+    output = p.stdout.read()
+    xcp.logger.info('pool_update.precheck %r returncode=%r output=%r', cmd, p.returncode, output)
```

Comment:
Thanks @lindig , For the is case(`yum install` a update_package), I also think its output should not big.